config/settings.toml

```diff
@@ -57,4 +57,4 @@ _remove_workspace = true
 ansible_extra_vars = ""
 
 # Choose how to install the operator. This can be "helm" or "kubectl"
-operator_installer = "helm"
+operator_installer = "kubectl"
```

Comment:
Might be worth noting that ceph_image only applies if operator_installer isn't helm.

Reply:
Again, soory. Committed my private changes...